helm/basex-validator/Chart.yaml

```diff
@@ -2,4 +2,4 @@ apiVersion: v1
 appVersion: latest
 description: BaseX Validator for validating XML against eLife's schematron
 name: basex-validator
-version: 0.1.7
+version: 0.1.8
```

Comment:
Is this worthy of a minor version bump? 0.2.0?

Reply:
It could be, I'll leave it up to you.